apps/webapp/src/state/staking.ts

```diff
@@ -158,21 +179,55 @@ export const createStakingSlice = (): SliceCreator<StakingSlice> => (set, get) =
       state.staking.votingPowerByValidatorInfo = getVotingPowerByValidatorInfo(validatorInfos);
     });
   },
-  loadUnstakedTokensByAccount: async () => {
+  loadUnstakedAndUnbondingTokensByAccount: async () => {
     const balancesByAccount = await getBalancesByAccount();
-    const unstakedTokensByAccount = balancesByAccount.reduce<Map<number, ValueView | undefined>>(
+    const { unstakedTokensByAccount, unbondingTokensByAccount } = balancesByAccount.reduce<{
```

Comment:
What do you think about splitting this up into two reduces? One for unstaked and one for unbonding? Feel it may be a bit more understandable as two helper functions.

Reply:
Ugh, yeah, I did it this way for performance reasons. If we split it into two `reduce`s, we'd have to make two passes over `balancesByAccount`, which _could_ be a pretty large array. 

Now that I write this, though, I realize most people will probably just use account #0 (right?), so this performance concern is probably an edge case. I'll split it into two helpers.

Reply:
[Done](https://github.com/penumbra-zone/web/pull/632/commits/8f90a65b42cb951fbf021d906136221a3676c551)